TombEngine/Objects/Effects/Boss.cpp

```diff
@@ -195,7 +195,7 @@ namespace TEN::Effects::Boss
 	}
 
 	// NOTE: Actual death occurs when countUntilDeath >= 60.
-	void ExplodeBoss(int itemNumber, ItemInfo& item, int countUntilDeath, const Vector4& color, bool allowExplosion)
+	void ExplodeBoss(int itemNumber, ItemInfo& item, int countUntilDeath, const Vector4& color, bool greenExplosion, bool allowExplosion)
```

Comment:
I am against introducing such specific arguments. What if I want to add `pinkExplosion` or `catsAndDogsExplosion` later? This must be changed.